wasm_runtime/src/c_api.rs

```diff
@@ -468,3 +500,26 @@ pub extern "C" fn wasm_executionctx_run(executionctx_id: *const c_char, _buffer:
 pub extern "C" fn wasm_return_const_char_ownership(ptr: *const c_char) {
     deallocate_cstring(ptr);
 }
+
+
+// ##########################################################################
+//                              Unit Tests
+// ##########################################################################
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::ffi::CString;
+
+    #[test]
+    fn test_wasm_config_entrypoint_set() {
```

Comment:
We are not prefixing the unit tests as `test_` since Rust enforces it to be located under mod tests. Also both IDE and `cargo` manages them well.

```suggestion
    fn wasm_config_entrypoint_set_general() {
```

Reply:
We usually add a suffix indicating what specific input are we testing (see examples at `ffi_utils.rs`).